lib/operations/admin_ops.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+const executeCommand = require('./db_ops').executeCommand;
+const executeDbAdminCommand = require('./db_ops').executeDbAdminCommand;
+const toError = require('../utils').toError;
+
+/**
+ * Execute the command using executeDbAdminCommand
+ *
+ * @param {Admin} a collection instance.
+ * @param {Object} command The command document
+ * @param {Object} [options] Optional settings. See Admin.prototype.serverStatus for a list of options.
+ * @param {Admin~resultCallback} [callback] The command result callback
+ */
+function constructCommand(admin, command, options, callback) {
+  executeDbAdminCommand(admin.s.db, command, options, (err, doc) => {
+    if (err == null && doc.ok === 1) return callback(null, doc);
+    if (err) return callback(err, false);
+    callback(toError(doc), false);
+  });
+}
+
+/**
+ * Get ReplicaSet status
+ *
+ * @param {Admin} a collection instance.
+ * @param {Object} [options] Optional settings. See Admin.prototype.replSetGetStatus for a list of options.
+ * @param {Admin~resultCallback} [callback] The command result callback.
+ */
+function replSetGetStatus(admin, options, callback) {
+  constructCommand(admin.s.db, { replSetGetStatus: 1 }, options, callback);
```

Comment:
Just curious why this command is passing `admin.s.db`, and the `serverStatus` passes just `admin`?

Reply:
That was definitely a mistake. Thanks for catching it!